### PR TITLE
fix(calendar-month): updated a11y per the react component

### PR DESCRIPTION
**Pull request by mcoker**
fixes https://github.com/patternfly/patternfly/issues/3679

> if a cell has .pf-m-selected applied you cannot tell when its button has :focus applied since the borders overlap

@mcarrano do you want to do something about this? We could use a blue shadow on focus or something like that?

Nov-20-2020 14-46-36

### CODE REVIEW

**Comment by mcarrano** [comment]
I think this is a good solution @mcoker .

File: src/patternfly/components/DatePicker/examples/DatePicker.md [hunk 1]
```diff
@@ -12,7 +12,12 @@ import './DatePicker.css'
 ### Basic
 ```hbs
 {{#> date-picker date-picker--id="basic"}}
-  {{> form-control controlType="input" input="true" form-control--modifier="pf-m-icon pf-m-calendar" form-control--attribute=(concat 'type="text" value="03/05/2020" id="' date-picker--id '-input" name="' date-picker--id '-input" aria-label="Basic date picker example"')}}
+  {{#> input-group}}
+    {{> form-control controlType="input" input="true" form-control--attribute=(concat 'type="text" value="03/05/2020" id="' date-picker--id '-input" name="' date-picker--id '-input" aria-label="Date picker"')}}
```
Comment (redallen) [inline src/patternfly/components/DatePicker/examples/DatePicker.md:16]: Can you change `value="03/05/2020"` to `value="2020-03-05"`? We also made that change upstream.

**Review by redallen — APPROVED** [review]
LGTM!!

**Review by mceledonia — APPROVED** [review]

**Review by mattnolting — APPROVED** [review]
LGTM! I tested w/VO and I the behavior is a little odd, but it's also consistent with google calendar and a few other JS calendars I tested. The `<td>` and `<button>` are focusable and announce different sets of info..

**Pull request merged**